packages/react-vapor/src/components/loading/components/ItemBoxLoading.tsx

```diff
@@ -8,13 +8,13 @@ const possibleWidths = ['mod-width-10', 'mod-width-20', 'mod-width-30', 'mod-wid
 export const ItemBoxLoading: React.FunctionComponent = () => (
     <li className="item-box disabled flex">
         <div
-            className={classNames('bg-grey-3 mod-rounded-border-1', Random.pick(possibleWidths))}
-            style={{height: '1em'}}
+            className={classNames('mod-rounded-border-1', Random.pick(possibleWidths))}
+            style={{height: '1em', backgroundColor: '#e5e8e8'}}
```

Comment:
IMO this style should be done somewhere in one of Vapor's scss file. That way we could have an overridable css variable for it.